File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;

class Solution {
    public String solution(String[] participant, String[] completion) {
        Arrays.sort(participant);
        Arrays.sort(completion);
        String s = "";
        for (int i = 0; i < participant.length; i++) {
            if (i == completion.length) {
                break;
            }
            
            if (!participant[i].equals(completion[i])) {
                s = participant[i];
                break;
            }
            s = participant[participant.length - 1];
        }
        return s;
    }
}

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    participant.sort()
    completion.sort()
    answer = ''

    for i in range(len(participant)):
        if i == len(completion): 
            answer = participant[i]
            break
        elif participant[i] != completion[i]:
            answer = participant[i]
            break
    
    return answer